lib/ListActions/ListActions.js

```diff
@@ -485,6 +485,8 @@ var ListActions = module.exports = kind(
 		var listAction, i;
 
 		this.listActionComponents = [];
+		this.$.listActions.destroyClientControls();
```

Comment:
Hi @kyo504, let's create a JIRA ticket for this so we can more easily track it, in case we need to figure out why we made that change, in the future. We can link it to the original ENYO-2218 issue.